ibc_public/utils_contrasts.py

```diff
@@ -1366,18 +1366,18 @@ def stop_nogo(design_matrix_columns):
 
 def finger_tapping(design_matrix_columns):
     """ Contrasts for finger tapping protocol"""
-    contrast_names = ['specified', 'chosen', 'null',
-                      'chosen-specified', 'specified-null', 'chosen-null']
+    contrast_names = ['specified', 'chosen', 'no_tap',
+                      'chosen-specified', 'specified-no_tap', 'chosen-no_tap']
     if design_matrix_columns is None:
         return dict([(name, []) for name in contrast_names])
     con = _elementary_contrasts(design_matrix_columns)
     contrasts = {
         'specified': con['specified'],
         'chosen': con['chosen'],
-        'null': con['null'],
+        'no_tap': con['no_tap'],
```

Comment:
I'm afraid that this is not going to work, since con has the strings in `design_matrix_columns` as keys.

Reply:
I think I addressed this [here](https://github.com/individual-brain-charting/analysis_pipeline/blob/master/ibc_main/neurospin_data/info/sub-04/camcan/ses-02/finger-tapping_sub-04_run-01.tsv) by renaming the entries of the event files as `no_tap`.

If I understood correctly, `make_paradigm` in [utils_paradigms](https://github.com/individual-brain-charting/public_analysis_code/blob/master/ibc_public/utils_paradigm.py) takes this file and gets the event info, and so far there are no adjustments needed in the  post_process function for this task. 

Is this what you meant ?